src/ScriptBuilder/Saga/Writers/PostgreSqlSagaScriptWriter.cs

```diff
@@ -53,8 +53,9 @@ from information_schema.columns
     public void WriteCreateIndex(CorrelationProperty correlationProperty)
     {
         writer.Write($@"
-createCorrelationIdx = 'create unique index if not exists ""Index_Correlation_{correlationProperty.Name}"" on public.""' || tableNameNonQuoted || '"" using btree (""Correlation_{correlationProperty.Name}"" asc);';
-execute createCorrelationIdx;");
+        script = 'create unique index if not exists ""Index_Correlation_{correlationProperty.Name}"" on public.""' || tableNameNonQuoted || '"" using btree (""Correlation_{correlationProperty.Name}"" asc);';
```

Comment:
I think the different name here was intentional, because that part will be wrapped in the "create table", but not 100% sure, we need to double check if that won't mess up the "outer" script variable value.

Reply:
I've checked and the correlation property creation part reused the `script` variable so I figured we can also reuse it here.